pkgs/applications/networking/whalebird/default.nix

```diff
@@ -0,0 +1,54 @@
+{ appimageTools, fetchurl, lib, gsettings-desktop-schemas, gtk3
+, makeDesktopItem
+}:
+
+let
+  pname = "whalebird";
+  version = "4.2.3";
+
+  icon = fetchurl {
+    url = "https://github.com/h3poteto/whalebird-desktop/blob/master/static/images/icon.png";
+    sha256 = "1d95v1zl0iizfav5g7wp1m8dldi2p4dfg3nsr7nr43qcm2l44h5s";
```

Comment:
```
copying path '/nix/store/3iggabd53y4j3hymwi8wbl828282lb8k-libav-11.12' from 'https://cache.nixos.org'...

trying https://github.com/h3poteto/whalebird-desktop/blob/master/static/images/icon.png
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
copying path '/nix/store/g5v8d14kyz5sbi5yxycfmrsrpnlkxh24-libdbusmenu-glib-16.04.0' from 'https://cache.nixos.org'...
100 83335    0 83335    0     0   217k      0 --:--:-- --:--:-- --:--:--  217k
hash mismatch in fixed-output derivation '/nix/store/zjdn5fd915vf7mbdbivgi1x9n6l2yhlx-icon.png':
  wanted: sha256:1d95v1zl0iizfav5g7wp1m8dldi2p4dfg3nsr7nr43qcm2l44h5s
  got:    sha256:0nj59bpzh8x8i438z1v5q5vwiqcqdxcmh2lbwi5wpzr11y8h73y8
copying path '/nix/store/gazv34zfz65ppkdv9zi29dgc11gvmdvr-libdbusmenu-gtk2-16.04.0' from 'https://cache.nixos.org'...
cannot build derivation '/nix/store/78gcak6qmidgmids71zpv3fk3p1a0fkr-whalebird.desktop.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/ckvffjqmkqpm6ws5cwikh8r32wzvb6zx-whalebird.drv': 1 dependencies couldn't be built
error: build of '/nix/store/ckvffjqmkqpm6ws5cwikh8r32wzvb6zx-whalebird.drv' failed
```

Reply:
Oh this is interesting! I had it build just fine maybe an hour ago.

Would it be ok if I place the icon next to default.nix? Or is this a bad idea?

Reply:
Result of `nixpkgs-review pr 101689` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>

  - whalebird
</details>